mama/c_cpp/src/c/subscription.c

```diff
@@ -1194,9 +1202,11 @@ mamaSubscription_getSubjectContext (mamaSubscription subscription,
         msgUtils_getIssueSymbol (msg, &issueSymbol);
         context->mSymbol = copyString (issueSymbol);
         #ifdef WITH_ENTITLEMENTS
+		{
```

Comment:
All of these WITH_ENTITLEMENTS changes provide Window compile compat. Same question as mempool.c